### ISSUE TITLE
When boarding friendly ships, you should have the option to help or attack.

### ISSUE THREAD

**Issue by mdsmestad** [endless-sky/endless-sky#651]
I noticed that when random battles occured, sometimes there would be disable ships after the fact. I think it should have the option to either assist or capture/raid these vessels even though you didn't attack them.

Another thing I noticed was there are never any derelict ships in the game. In EV sometimes you would find derelict ships in space (though they were sometimes traps set by pirates) and you could board and possibly capture/and or raid them. I'd love to see this implemented. It's very satisfying to find random vessels adrift. Kinda like finding money left lying around.

**Comment by Thunderforge** [comment]
I think that adding derelict ships (including derelict decoys) should be a separate issue.

Regarding the option to help or attack, I think that right now you default to helping them if you board them by offering repairs. Are you wanting an option to raid them instead?

**Comment by mdsmestad** [comment]
Yes.

**Comment by thenerdfreak** [comment]
I agree. sometimes while I'm playing vulture tactics, a friendly ship that is disabled is more desirable than the hostiles, and I want to take it and run off regardless of consequences

**Comment by Thunderforge** [comment]
Perhaps the boarding mission could be modified? Instead of the captain thanking you and handing over some credits, maybe they ask you if you are there to help repair the ship. If you say yes, they thank you and give you a reward. If you say no, then the normal boarding procedure happens instead.

**Comment by bobafetthotmail** [comment]
Would make sense only if you could capture friendly ships without having to murder all their crew with standard boarding.
Something like "I save you but I want your ship as payment".

Otherwise this mechanic is only going to be used ONCE, then the faction of the "friendly" ship will hate you. (because killing off 30-100 people to take a ship isn't going to look good, y'know)

I think that if you shoot at other ships of the same faction to get them to aggro on you, then you can go and cap the disabled one.

**Comment by endless-sky** [comment]
The game mechanics already mostly exist to create derelict ships - they can be spawned by an invisible mission and trigger some response when you board them.

**Comment by thenerdfreak** [comment]
It would be cool to have a few ships only obtainable through finding them direlect in completely uninhabited strings of systems. Like remnants of an alien species that literally doesn't exist anymore. Should be extremely hard to get there though.

**Comment by DingusShingleton** [comment]
Ah, another great idea for the uninhabited region of space as mentioned in #612. In addition to derelict-only ships, it would also be a cool way to introduce a very small chance of getting other rare ships, like korath, pug, or even quarg. Purely based on chance, for instance, a 1% chance that a system in that region contains a derelict ship, and then a very small chance on top of that of it being something really valuable. And then you had better hope you have enough crew or cargo space to take the ship, or at least its outfits.

It would also be a cool way to trigger missions, like discovering a ship so badly damaged that it can't be captured, but still contains some long lost alien tech that you can deliver to your buddies at Tarazed (or the syndicate? Perhaps igniting a bidding war?) for a new awesome outfit.

**Comment by BuildBruh** [comment]
I actually have a derelict ship plugin in the works, but the spawn rates are something I'm puzzled on.

I currently have it at 10% for all ships, with a 70% chance of that to spawn fighters/interceptors/(cheaper) transports. 20% for heavy freighters, rest of transports and light warships. 6% medium warships, 3.75% chance of heavy warships, and a .25% chance for a derelict human-ish worldship to spawn in worlds within a 4 jump radius of Deep, and over the rainbow. (This is mostly due to us knowing that the **SPOILERS** pug are able to control wormholes and have a history of meddling with nearly all species that have existed so far.)

**Comment by Startingfresher** [comment]
This thread definitely counts as necro'ed, but since it isn't closed, I'd like to say that I'd really be interested in an option to repair enemy ships, which would make them non-hostile(to you) and attempt to leave the system.

I would assume this would also come with some small reduction of the negative rep from disabling the ship, particularly with hostile factions (you are at war, a stolen outfit/cargo here or there isn't worth bringing up imo), though I definitely wouldn't recommend it gives a positive total rep for anti-farming reasons. I personally am not really invested in the reputation change idea, this is mostly for RP purposes. 

If this behavior were implemented in to NPCs, I'd expect it to be used mainly by pirates/korath/the remanent, stealing enemy cargo/outfits and repairing them enough to escape, in order to encourage risky(profitable) behavior on behalf of their targets. It also would do abit of work in explaining the sheer quantity of ships in the galaxy, even with constant battles with pirates.
